### DISCUSSION TITLE
Table in Blazor do not show pageSizeChanger

### DISCUSSION THREAD

**Discussion by ghost** [ant-design-blazor/ant-design-blazor#2357] [Q&A] [answered]
When table has many pages, then pagination in bottom Table is with pageSizeChanger (10/20/50/100 etc.)
When table has only 2 pages (18 records = first page 10 rec and second has 8 records), but pageSizeChanger do not showing ?

How activate pageSizeChanger for allways?
BLAZOR SERVER!

Thanks for advice!

**Comment by lukblazewicz** [comment, marked as answer]
@JosefBoska 
If I remember correctly, there is an automatic mechanism that hides pagination if some minimal amount of items is not reached. We bypassed this behaviour by using `<PaginationTemplate>` inside `<AntDesing.Table>` and passing there a `<Pagination>` component with `ShowSizeChanger` parameter set to `true`.

**Reply by anranruye** [reply]
and here is the demo: https://antblazor.com/en-US/components/table#components-table-demo-custom-pagination
